README.md

```diff
@@ -395,3 +397,34 @@ The [ss-open/ci/recipes project](https://gitlab.com/ss-open/ci/recipes) offers a
 
 If you have any questions, suggestions, need a wrapper for a programming language or just want to chat join #editorconfig-checker on freenode(IRC).
 If you don't have an IRC-client set up you can use the [freenode webchat](https://webchat.freenode.net/?channels=editorconfig-checker).
+
+## Contributing
+
+Anyone can help to improve the project, submit a Feature Request, a bug report or even correct a spelling mistake.
+
+The steps to contribute can be found in the [CONTRIBUTING.md](./CONTRIBUTING.md) file.
+
+## Semantic Versioning Policy
+
+**editorconfig-checker** adheres to [Semantic Versioning](https://semver.org/) for releases.
+
+However, as it is a code quality tool, it's not always clear when a minor or major version bump occurs. The following rules are used to determine the version bump:
+
+- Patch release (1.0.x -> 1.0.y)
+  - Updates to output formats (error messages, logs, ...).
+  - Performance improvements which doesn't affect behavior.
+  - Build process changes (e.g., updating dependencies, updating `Dockerfile`, ...).
+  - Reverts (reverting a previous commit).
+  - Bug fixes which result in **editorconfig-checker** reporting less linting errors (removing "false-positive" linting errors).
+- Minor release (1.x.0 -> 1.y.0)
+  - Adding new [configuration options](#configuration), including new CLI flags.
+  - Adding new [path to exclude by default](#default-excludes).
+  - Adding new [output formats](#formats).
+  - Supporting a new [editorconfig](https://editorconfig.org/) property (e.g: `insert_final_newline`, `indent_size`, ...).
+  - Any new feature which doesn't break existing behavior.
+- Major release (x.0.0 -> y.0.0)
+  - Removal of a [configuration](#configuration) option.
+  - Removal of an [output format](#formats).
+  - Removal of a [path to exclude by default](#default-excludes).
+  - Removal of support for an [editorconfig](https://editorconfig.org/) property.
+  - Bug fixes, which might result in **editorconfig-checker** reporting more linting errors, if the previous behavior was incorrect, with the [editorconfig](https://editorconfig.org/) specification.
```

Comment:
Okay, than I think this might communicate this intend better:
```suggestion
  - Bug fixes, which result in **editorconfig-checker** reporting more linting errors, even if the previous behavior was incorrect according to the [editorconfig specification](https://editorconfig.org/).
```

Reply:
I'm not convinced by "even if", it's more like "because" ... the previous behavior was incorrect.

Reply:
I've updated it to: `Bug fixes, which result in **editorconfig-checker** reporting more linting errors, because the previous behavior was incorrect according to the [editorconfig specification](https://editorconfig.org/)`.